feature-libs/product/configuration/src/util/serialization-utils.ts

```diff
@@ -0,0 +1,52 @@
+import { HttpErrorResponse } from '@angular/common/http';
+import { ErrorModel, HttpErrorModel } from '@spartacus/core';
+
+export const UNKNOWN_ERROR = {
```

Comment:
Red flag for us :)
I understand you copy-pasted the code because this is not exposed.

Reply:
Correct :-) 
I added the respective export to  /projects/core/src/util/index.ts